### DISCUSSION TITLE
How to check if an user is in specific group

### DISCUSSION THREAD

**Discussion by klapsius** [DirectoryTree/LdapRecord#606] [Q&A]
In my Laravel application, I use Scope to check if a user is in a group, but it does not work.
If I try: `$query->in('cn=My-App-Login,OU=SecurityGroups,DC=my_domain,DC=com');` it does not work
if I try `$query->in('DC=my_domain,DC=com')` it allows log-in to any user in `AD`.
Do I need to use something different?

**Comment by stevebauman** [comment]
Hi @klapsius,

You cannot scope results in that way using a Group's distinguished name, only Organization Unit's.

Please see this documentation:

https://ldaprecord.com/docs/core/v3/active-directory/users#checking-existence
